srcpkgs/lean-community/template

```diff
@@ -0,0 +1,17 @@
+# Template file for 'lean-community'
+pkgname=lean-community
+version=3.32.1
+revision=1
+wrksrc="lean-$version"
+build_wrksrc=src
+build_style=cmake
+make_cmd=ninja
+hostmakedepends="cmake ninja gmp"
+makedepends="gmp-devel"
+depends="gmp"
```

Comment:
The runtime dependency on `gmp`  is not needed, it should be automatic based on sonames.

Reply:
Thanks, I removed it.